OloEngine/src/OloEngine/Audio/SoundGraph/SoundGraphSound.cpp

```
@@ -75,6 +93,16 @@ namespace OloEngine::Audio::SoundGraph
         return true;
     }
 
+    bool SoundGraphSound::InitializeDetachedSource()
```

Cppcheck (2.21.0)

[style] The function 'InitializeDetachedSource' is never used.

(unusedFunction)